test/Microsoft.TestPlatform.AcceptanceTests/DataCollectionTests.cs

```diff
@@ -50,7 +52,15 @@ private string GetRunsettingsFilePath()
                 this.testEnvironment.RunnerFramework,
                 "OutOfProcDataCollector.dll");
 
-            dataCollectionAttributes.Add("assemblyQualifiedName", "OutOfProcDataCollector.SampleDataCollector, OutOfProcDataCollector, Version=15.0.0.0, Culture=neutral, PublicKeyToken=null");
+            var outOfProcAssemblyPath = this.testEnvironment.GetTestAsset("OutOfProcDataCollector.dll");
+
+#if !NET46
+            var assemblyName = AssemblyLoadContext.GetAssemblyName(outOfProcAssemblyPath);
+#else
+            var assemblyName = AssemblyName.GetAssemblyName(outOfProcAssemblyPath);
+#endif
+
+            dataCollectionAttributes.Add("assemblyQualifiedName", string.Format("OutOfProcDataCollector.SampleDataCollector, {0}", assemblyName));
```

Comment:
Do we need an assert that attachment is actually copied to ResultsDirectory? Or was the test case already failing and with the product fix it is green now?